Cabal/Distribution/Simple/Configure.hs

```diff
@@ -489,8 +535,10 @@ configure (pkg_descr0', pbi) cfg = do
          ++ "supported by " ++ display (compilerId comp) ++ ": "
          ++ intercalate ", " (map display exts)
 
-    -- configured known/required programs & external build tools
-    -- exclude build-tool deps on "internal" exes in the same package
+    -- Configure known/required programs & external build tools.
+    -- Exclude build-tool deps on "internal" exes in the same package
+    --
+    -- ToDo: Factor this into a helper package.
```

Comment:
`TODO`s in existing code are written in all caps: let's be consistent, it makes them easier to grep for.